### PR TITLE
add Huobi Eco Chain testnet

### PR DESCRIPTION
**Pull request by somereason**
Add configuration for Huobi Eco Chain testnet, [https://www.hecochain.com](https://www.hecochain.com)

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -202,5 +202,19 @@
...
+    "shortName": "heco",
+    "chainId": 256,
+    "network": "testnet",
+    "rpc": [
+      "https://http-testnet.hecochain.com"
+    ],
+    "ws": [
+      "wss://ws-testnet.hecochain.com"
+    ],
+    "explorer": "http://scan-testnet.hecochain.com/"
```
Comment (bonustrack) [inline src/networks.json:218]: Can you change this
```suggestion
    "explorer": "https://scan-testnet.hecochain.com"
```

File: src/networks.json [hunk 2]
```diff
@@ -202,5 +202,19 @@
...
     "explorer": "https://scan.thundercore.com"
+  },
+  "256": {
+    "key": "256",
+    "name": "Huobi Eco Chain Testnet",
+    "shortName": "heco",
+    "chainId": 256,
+    "network": "testnet",
+    "rpc": [
+      "https://http-testnet.hecochain.com"
```
Comment (bonustrack) [inline src/networks.json:213]: Is this an archive node?

Comment (somereason) [reply src/networks.json:213]: sure, try this

curl --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}' -H "content-type:application/json" https://http-testnet.hecochain.com

**Review by bonustrack — APPROVED** [review]
Looks good!

**Pull request merged**